components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -150,12 +150,7 @@ func init() {
 }
 
 // Note: the indentation is to keep this in line with writer.Body()
-const alreadyMigratedMessage = `You have already upgraded to IAM %s.
-  If you wish to re-run the migration, first run:
-
-      chef-automate iam reset-to-v1
-
-  Then re-run this command.`
+const alreadyMigratedMessage = `You are already using IAM %s.`
```

Comment:
nit: something sounds weird about "using" a version. What do you think about `"You are already on IAM version %s."`?

Reply:
I support this change!

Reply:
I support this change!